openshift/template.yaml

```diff
@@ -38,6 +38,8 @@ objects:
             value: bayesian-api
           - name: COREAPI_SERVICE_PORT
             value: "5000"
+          - name: FLASK_LOGGING_LEVEL
+            value: ${FLASK_LOGGING_LEVEL}
```

Comment:
@GeetikaBatra FLASK_LOGGING_LEVEL has to be defined with a value in the parameter section. [Here is an example](https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/openshift/template.yaml#L185-L189) of how to do that.